src/OpenLoco/src/Objects/RoadObject.h

```diff
@@ -16,42 +17,46 @@ namespace OpenLoco
         struct RenderTarget;
     }
 
-    namespace Flags12
+    enum class RoadObjectFlags : uint16_t // Made this a uint16_t to maintain the size of the RoadObject struct
```

Comment:
Sorry I missed this, you can remove that comment also, I believe the flags were defined incorrectly. Other than that the rest looks good to me.